### PR TITLE
LPS-194351 | Edit and Delete generic creation button actions

### PR DESCRIPTION
**Pull request by andreluis2022**
**Description:**
Create tests Product QA | Functional Automation - LPS-194351 - Edit and Delete generic creation button actions

JIRA Ticket:
https://liferay.atlassian.net/browse/LPS-194351

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete can you take a look at these changes?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+		task ("Then the Delete and Edit option is listed in the dropdown menu") {
+			AssertElementPresent(
+				key_menuItem = "Delete",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be deleted, and a success message is displayed"
+	@priority = 5
+	test CanBeDeleteAction {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:116]: This is a priority 5 test, so you have to add the `property portal.acceptance = "true";`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be edited and changes are displayed"
+	@priority = 5
+	test CanConfirmChangesInActionCreated {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:247]: Change the name to something more simple: `CanEditCreationAction` is easier to understand

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+		task ("And it appears on the creation Action tab table with the original label.") {
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be edited and changes are displayed"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:246]: This is a priority 5 test, so you have to add the `property portal.acceptance = "true";`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 4]
```diff
@@ -0,0 +1,526 @@
...
+				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE");
+
+			AssertElementPresent(
+				key_unnamedTitle = "",
+				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that when editing an already created Link action, it is not possible to leave the fields empty"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:393]: Low this priority to 4 (I've already modified it in the story)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 5]
```diff
@@ -0,0 +1,526 @@
...
+			AssertTextEquals(
+				locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+				value1 = "This field is required.");
+
+			AssertElementPresent(locator1 = "DataSet#ANY_BUTTON_DISABLED");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that when editing an already created modal action, it is not possible to leave the fields empty"
+	@priority = 5
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:438]: Low this priority to 4 (I've already modified it in the story)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase [hunk 6]
```diff
@@ -0,0 +1,526 @@
...
+				locator1 = "Message#ERROR_FORM_FIELD_REQUIRED",
+				value1 = "This field is required.");
+
+			AssertElementPresent(locator1 = "DataSet#ANY_BUTTON_DISABLED");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be edited, and options are displayed"
+	@priority = 3
+	test TheNewActionCanBeEdited {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase:490]: You can remove this test. I've lowered this scenario to 2 in the story, so no automation test is needed.

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]
Take a look at my request messages :)

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete what do you think ?

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**